server.rb

```diff
@@ -0,0 +1,189 @@
+require "sinatra/base"
+require "sqlite3"
+require "nokogiri"
+require 'active_support'
+require 'json'
+require 'securerandom'
+
+class Server < Sinatra::Base
+
+  # {
+  #  to: <UUID>
+  #  from: <UUID>
+  #  message: <String>
+  # }
+
+  post '/messages/:format?' do
+    case params['format']
+    when 'json' then
+      message = JSONMessage.new(request.body.read)
+      if message.valid?
+        id = message.save
```

Comment:
We should check the success of `save`. Also, see below for ideas about grouping `save` and `send_message`.